packages/devtools-reps/src/object-inspector/component.js

```diff
@@ -165,13 +174,45 @@ class ObjectInspector extends Component<Props> {
   }
 
   getNodeKey(item: Node): string {
-    return item.path && typeof item.path.toString === "function"
-      ? item.path.toString()
-      : JSON.stringify(item);
+    let key =
+      item.path && typeof item.path.toString === "function"
+        ? item.path.toString()
+        : JSON.stringify(item);
+
+    const { evaluations } = this.props;
+    const evaluation = evaluations.get(item.path);
+    if (evaluation) {
+      key = `${key}-${evaluation.timestamp}`;
```

Comment:
we need to change the key for the node to be re-rendered (i made the change to the key to be `${key}-evaluated`)

Reply:
that's lame... where is that check?

Reply:
So, that's not really accurate, having a different key allows to bypass the `shouldComponentUpdate` done in the `TreeNode` component: 

```js
  shouldComponentUpdate(nextProps) {
    return (
      this.props.item !== nextProps.item ||
      this.props.focused !== nextProps.focused ||
      this.props.expanded !== nextProps.expanded
    );
  }
```

So in the case where we only evaluate a node, we don't modify the item, nor we focus or expand it, so it does not update.
I guess it would be better to change the item.